src/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -2852,6 +2855,9 @@
   </data>
   <data name="XPath_UnexpectedToken" xml:space="preserve">
     <value>Unexpected token '{0}' in the expression.</value>
+  </data>
+    <data name="XPath_UnexpectedToken2" xml:space="preserve">
```

Comment:
If this is fixable by the user then leave it if not then I'd go with reusing vague InternalError and add comment in the code

Reply:
done